x-pack/plugins/lens/public/embeddable/embeddable.tsx

```diff
@@ -344,6 +344,10 @@ export class Embeddable
     );
   }
 
+  public getVisualizationType() {
```

Comment:
@kibana-vis-editors I needed to expose visualization type in `synchronizeMovementAction` action so that the action is displayed for lens regionmap visualizations.